src/data/cel.rs

```diff
@@ -296,11 +309,11 @@ fn json_to_cel(json: &str) -> Value {
             JsonValue::Bool(b) => b.into(),
             JsonValue::Number(n) => {
                 if n.is_u64() {
-                    n.as_u64().unwrap().into()
+                    n.as_u64().expect("number must be u64").into()
```

Comment:
You could maybe prefix that string with `Unreachable: `, as the contract is that if `.is_<T>`, `.as_<T>` succeeds

Reply:
Yeah that's clearer, that was kinda what I was trying to convey in my message.. that the number cannot be anything other than the type